exonum/src/lib.rs

```diff
@@ -18,6 +18,15 @@
 
 #![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy_pedantic))]
+#![cfg_attr(
+    feature = "cargo-clippy",
+    allow(
+          stutter, similar_names, items_after_statements, use_self,
+          cast_possible_wrap, cast_possible_truncation, cast_sign_loss,
+          pub_enum_variant_names, single_match_else, option_map_unwrap_or_else
```

Comment:
`pub_enum_variant_names` can be controversial (however I'm curious about how many such cases do we have), but why are `single_match_else` and `option_map_unwrap_or_else` suppressed?